style_editor.html

```diff
@@ -8,14 +8,167 @@
   <title>ProffieOS Style Editor</title>
 <style>
 
-.button_on {
-  background: lightblue;
+/* Make page elements non-selectable with drag box */
+/* Took a vote, enabled won, but disabled for now. */
+/** {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}*/
+
+/* Box sizing helps keep buttons uniform in
+size when padding and margins may change */
+* {
+  box-sizing: border-box;
+}
+
+/* Submit, copy, etc... */
+.middle-btn-container {
+  white-space: nowrap;
+}
+/* Expand and Layerize buttons */
+.middle-btn-container .button-off,
+.middle-btn-container .button-off:hover {
+  background-color: lightgray;
+  color: darkgray;
+}
+.middle-btn-container .button-on {
+  padding: 2px;
+  border: 2px solid dodgerblue;
+}
+/* Other buttons */
+button,
+input[type="button"] {
+  background-color: #ccc;
+  color: black;
+  border-radius: 2px;
+  padding: 3px;
+  font-size: 14px;
+  border: 1px solid #bbb;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-weight: normal;
+}
+button:hover,
+select:hover,
+input[type="button"]:hover {
+  background-color: #999;
```

Comment:
It seems weird to make the background middle gray, why?

Reply:
It's just a hover-over-button color. Gives a tactile feel.
How about just lighting up the border instead?

Reply:
Maybe?
It's hard to imagine exactly what it looks like.

Reply:
> Maybe? It's hard to imagine exactly what it looks like.

https://nosloppy.github.io/ProffieOS-StyleEditor-1/style_editor_BC_V5.html